src/components/TaskItem.tsx

```diff
@@ -15,9 +21,13 @@ export default function TaskItem({ taskName }: TaskItemProps) {
       </div>
       <div className="btn-group">
         <button type="button" className="btn">
-          Save <span className="visually-hidden" />
+          Edit <span className="visually-hidden" />
         </button>
-        <button type="button" className="btn btn__danger">
+        <button
+          type="button"
+          className="btn btn__danger"
+          onClick={() => deleteTask(index)}
```

Comment:
se esta usando  index qual a necessidade de id?

Reply:
so uma obs , falto dar deploy no ghpages tmb

Reply:
entendi nao

Reply:
github pages ta na versao sem essa funcionalidade tendeu

Reply:
Se seguir meu comment acima, aqui ficaria

```suggestion
          onClick={deleteTask}
```

Daria uma simplificada legal! 😁